lib/rspec/core/metadata.rb

```diff
@@ -326,7 +275,7 @@ def full_description
         :description,
         :description_args,
         :described_class,
-        :example_group,
+        :example_group, # FIXME: should we also remove this?
```

Comment:
Highlight. Should we remove this?

Reply:
I don't think so, we still set it and can be used to retrieve data, so I think it needs to stay reserved to preserve any down stream behaviour that uses metadata for other purposes?